gradle.properties

```diff
@@ -0,0 +1,2 @@
+# For standalone build
+standaloneGradlePluginsVersion=1.27.0_standaloneBuild-SNAPSHOT
```

Comment:
Why not use `gradlePluginsVersion` here?

Reply:
I was trying to avoid overwriting properties in a standard build but, on second thought, I don't think the standard build even cares about `gradlePluginsVersion` in the module project. I removed the special property.